examples/agent/websocket/no_mic/main.go

```diff
@@ -4,7 +4,7 @@
 
 package main
 
-// streaming
+// Import dependencies
```

Comment:
My changes:

- add a chatlog function to print interactions
- re-organizes code so it follows the guide structure more closely
- adds helpful comments.

Note: I'm still not able to get the agent to response after the initial welcome message, though at one point that worked for me, @naomi-lgbt  I need to spend a bit more time on it